workers_kv.go

```diff
@@ -82,19 +83,39 @@ func (api *API) CreateWorkersKVNamespace(ctx context.Context, req *WorkersKVName
 // ListWorkersKVNamespaces lists storage namespaces
 //
 // API reference: https://api.cloudflare.com/#workers-kv-namespace-list-namespaces
-func (api *API) ListWorkersKVNamespaces(ctx context.Context) (ListWorkersKVNamespacesResponse, error) {
-	uri := fmt.Sprintf("/accounts/%s/storage/kv/namespaces", api.AccountID)
-	res, err := api.makeRequestContext(ctx, http.MethodGet, uri, nil)
-	if err != nil {
-		return ListWorkersKVNamespacesResponse{}, errors.Wrap(err, errMakeRequestError)
-	}
+func (api *API) ListWorkersKVNamespaces() ([]WorkersKVNamespace, error) {
```

Comment:
I've changed the signature of this function, which is a breaking change for the library, after the discussion on the first iteration of the PR. Is this acceptable? Now this function is a bit unique in workers_kv.go, but it's a cleaner implementation than my first attempt, and, IMO, more useful functionality for most users of the library.

Reply:
I'd keep context, reasoning in comment below. Changing return type is fine.